tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -534,23 +571,36 @@ class LineChart {
   }
 
   private resetXDomain() {
-    // (Copied from DragZoomLayer.unzoom.)
-    const xScale = this.xScale as any;
-    xScale._domainMin = null;
-    xScale._domainMax = null;
-    const xDomain = xScale._getExtent();
+    let xDomain;
+    if (this._defaultXRange) {
+      // Use the range specified by the caller.
```

Comment:
nitpick: as specified by the chart constructor (I would view the "caller" as the call to resetXDomain, and there is no range specification in that call)

Reply:
True - how about `higher-level component`?